### ISSUE TITLE
bug that you cannot use commands 1.19.2

### ISSUE THREAD

**Issue by papillo12** [Meridanus/fabric_xp_storage_additions#18]
hello, sorry for not having a crash report or a log is that it was long ago that I solved the problem, but when you enter a server and try to use commands from the game does not let you use them, it was until I did a discard mods that I found the problem was your mod

**Comment by Meridanus** [comment]
Does the error occur with only my mods present? I can't find any command related bug from my side.

**Comment by papillo12** [comment]
hello, yes, you could use commands from the server panel, but inside the game it was no longer possible

**Comment by Meridanus** [comment]
That's very Strange, when i test this with Fabric, Xp Obelisk and Xp Obelisk Additions, there's no problem with in game Commands.

Can you tell me more about your server setup?! can't reproduce the bug on my system.

**Comment by jarocajavi** [comment]
same for me

https://pastebin.com/HZB7R8gd

when i try type a command get a strange bug than tells a problem with fabric

https://i.imgur.com/Gt3DsWb.png

**Comment by papillo12** [comment]
> same for me
> 
> https://pastebin.com/HZB7R8gd
> 
> when i try type a command get a strange bug than tells a problem with fabric
> 
> https://i.imgur.com/Gt3DsWb.png

that's exactly the error I had

**Comment by Meridanus** [comment]
I didn't use the ClientCommandManager on my mod see: https://docproject.github.io/fabricmc_fabric/net/fabricmc/fabric/api/client/command/v1/ClientCommandManager.html

I have no idea what happens... and can't reproduce the bug. I think its some other mod that causes this error.

**Comment by kevinthegreat1** [comment]
mod xps_additions is throwing an exception in its listener registered to `ClientPlayConnectionEvents.JOIN`, which causes `ClientPlayNetworkHandler.onGameJoin()` to exit early which causes `ClientCommandInternals.activeDispatcher` to not be set. Then, running a command from the client will throw a null pointer exception as `activeDispatcher` is not set.

Attached is `.mixin.out` and the relevant log.
[mixin.out.zip](https://github.com/Meridanus/fabric_xp_storage_additions/files/10302356/mixin.out.zip)
[latest.log](https://github.com/Meridanus/fabric_xp_storage_additions/files/10302359/latest.log)

**Comment by Meridanus** [comment]
That's good to know. But i have no idea how to fix that at the moment :/ cant reproduce this bug on my IDE.

something here goes wrong:

```
 ClientPlayConnectionEvents.JOIN.register((packetListener, packetSender, mc) -> {
            if (StaffOfRebark.STRIPPED_BLOCKS == null) {
                System.out.println("Null on Client");
                StaffOfRebark.STRIPPED_BLOCKS = AxeItemAccessor.getStrip().entrySet().stream()
                .map((e) -> new AbstractMap.SimpleImmutableEntry<>(e.getKey(), e.getValue()))
                .collect(Collectors.toMap(Map.Entry::getValue, Map.Entry::getKey));
            }
        });
```

**Comment by Meridanus** [comment]
Fixed wit the last commit, Mod version 0.1.8
